helmfile/charts/gatekeeper/podsecuritypolicies/templates/custom/mutations/privilege-escalation.yaml

```
@@ -0,0 +1,63 @@
+{{- range $namespace, $services := .Values.constraints }}
+{{- range $name, $value := $services }}
+{{- if get $value.allow "allowPrivilegeEscalation" }}
+---
+apiVersion: mutations.gatekeeper.sh/v1
+kind: Assign
+metadata:
+  name: {{ printf "privilege-escalation-%s" $name | trunc 63 }}
```

The way I've done with the rest is:
```suggestion
  name: {{ printf "privilege-escalation-%s-container" $name | trunc 63 }}
```
And then the same with `ephemeral` and `init` so each are grouped together based on type and then on service.

---

I have updated, PTAL